packages/terra-popup/tests/wdio/popup-spec.js

```diff
@@ -0,0 +1,17 @@
+/* global  browser,Terra */
+const viewports = Terra.viewports('tiny', 'medium');
+
+describe('Popup', () => {
+  describe('Arrow Popup', () => {
+    beforeEach(() => browser.url('/#/tests/popup-tests/arrow'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-popup-content-inner-border': '10px dashed red',
+      '--terra-popup-content-rounded-corners-border-radius': '20px',
```

Comment:
Remove this tests since it's no longer themeable.

Reply:
[Updated](https://github.com/cerner/terra-core/pull/1214/commits/fef76612878adc5e91414cb7736aa2605d87cb9a#diff-75a22fd845592219495fcc77ea6104c3R10).